.rubocop.yml

```diff
@@ -7,3 +7,6 @@ AllCops:
 
 ParameterLists:
   CountKeywordArgs: false
+
+Metrics/ModuleLength:
```

Comment:
Since coverage is provided by the up-to-date CocoaPods config should we squash lines 10-11?